### PR TITLE
SpringBoot 3.0: Rename bundles and features

### PR DESCRIPTION
**Pull request by cbridgha**

### CODE REVIEW

**Review by hlhoots — DISMISSED** [review]
LGTM pending a good build.

**Comment by posmith** [comment]
L2 message review not required

**Comment by cbridgha** [comment]
#build

File: dev/com.ibm.ws.springboot.support.shutdown/src/com/ibm/ws/springboot/support/shutdown/FeatureAuditor.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ public void postProcessEnvironment(ConfigurableEnvironment env, SpringApplicatio
         try {
             // Not sure if this is correct....
             Class.forName("org.springframework.boot.web.servlet.server.ServletWebServerFactory");
-            Class.forName("com.ibm.ws.springboot.support.web.server.version30.container.LibertyConfiguration");
+            Class.forName("io.openliberty.springboot.support.web.server.version30.container.LibertyConfiguration");
```
Comment (jhanders34) [inline dev/com.ibm.ws.springboot.support.shutdown/src/com/ibm/ws/springboot/support/shutdown/FeatureAuditor.java:70]: How is LibertyConfiguration loadable if it is not exported by the springboot.support bundle?

**Review by jhanders34 — APPROVED** [review]

**Comment by jhanders34** [comment]
#libby

**Comment by cbridgha** [comment]
Build with changes: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_xdfD0AkREe6txPsqA1YBCw

File: dev/com.ibm.ws.springboot.support.shutdown/src/com/ibm/ws/springboot/support/shutdown/FeatureAuditor.java (hunk 1 above)
Comment (cbridgha) [reply dev/com.ibm.ws.springboot.support.shutdown/src/com/ibm/ws/springboot/support/shutdown/FeatureAuditor.java:70]: A service ref declared in com.ibm.ws.app.manager.springboot is resolved to  springBoot30Server.jar, and those packages are declared private but available to that FeatureAuditor class

**Pull request merged**